chapters/gamestructure.adoc

```diff
@@ -96,6 +96,11 @@ The <<Referee, referee>> asks the <<Robot Handler, robot handlers>> of the teams
 
 However, if both teams prefer the same color, the referee assigns the colors by chance. In this case, the teams switch the colors after the first half of the match as well as after the first half of the overtime if applicable.
 
+==== Choosing Hulls Colors
```

Comment:
Shouldn't it be "Choosing Hull Colors"? Or more explicit "Choosing Robot Hull Colors". Or "Choosing Color of the Robot Hulls".

Reply:
Yep, I changed for the explicit one.